src/pages/dashboard/dashboardComponenets/recentMessage.tsx

```diff
@@ -1,20 +1,58 @@
-import React from "react";
-import DateTime from "./dateTime";
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { getInquiries } from "../../../store/inquiry-store/inquiryAction";
+import { IInquiry } from "../../../interfaces";
 
 const RecentMessage: React.FC = () => {
+
+	const dispatch = useDispatch();
+	const state = useSelector((state) => state.inquiryReducer);
+	const [inquiries, setInquiries] = useState<IInquiry[]>([]);
+
+	// Fetch inquiries information
+	useEffect(() => {
+		dispatch(getInquiries());
+	}, [dispatch, getInquiries]);
+
+	// Set Inquiry data
+	useEffect(() => {
+		setInquiries(state.inquiries);
+	}, [state.inquiries, setInquiries]);
+
+	const ListItem = () =>{
+		return (
+			<div>
+				{
+				inquiries.slice(0,3).map((inquiry)=>{
+					return(
+						<>
+							<div className="dateTime d-flex">
+								<div className="col-sm-6">
+									<p>{new Date(inquiry?.createdAt).toLocaleTimeString("en-US")}</p>
+									<p>{new Date(inquiry?.createdAt).toLocaleDateString("en-US")}</p>
+								</div>
+								<div className="col-sm"></div>
+								<div className="col-sm-5">
+									<p>{inquiry?.name}</p>
+									<p>User</p>
```

Comment:
Remove this line. It's not necessary. And after removing this, the vertical alignment of DateTime and Name should be middle.

Reply:
changes are added